frontend/src/components/page-main/upcoming.js

```diff
@@ -10,7 +10,7 @@ const UpcomingEvents = () => (
         alt="UpcomingEventsImg"
       />
     </div>
-    <div className="w-full lg:w-2/4 text-center lg:text-left">
+    <div className="w-full lg:2/4 text-center lg:text-left">
```

Comment:
why is the 'w' removed? i looked online at some documentation but could not find examples of lg:2/4 being done. looking at the screen itself, the width is not 50%.

<img width="1440" alt="Screenshot 2024-04-07 at 6 16 13 PM" src="https://github.com/hmcc-global/hmccaa-web/assets/88253868/7387767e-c5d0-40f7-8cf4-1eae63dcca22">

Reply:
on the current main branch 4/7/24 (todays date) it is centered. theres probably some conflicts you need to resolve with main